opba-protocols/xs2a-protocol/src/main/java/de/adorsys/opba/protocol/xs2a/service/xs2a/validation/Xs2aValidator.java

```diff
@@ -97,6 +95,19 @@ private ValidationIssue toIssue(ConstraintViolation<Object> violation) {
                 .build();
     }
 
+    private boolean doNotIgnoreValidationError(ConstraintViolation<Object> constraint, Map<FieldCode, IgnoreValidationRule> rulesMap) {
+        ValidationInfo info = findInfoOnViolation(constraint);
+        boolean doNotIgnoreValidationError = info.validationMode() == ValidationMode.MANDATORY;
+
+        FieldCode fieldCode = info.ctx().value();
+        IgnoreValidationRule ignoreRule = rulesMap.get(fieldCode);
+        if (ignoreRule != null) {
+            doNotIgnoreValidationError = ignoreRule.apply();
```

Comment:
Still, it is better to name `apply()` as `applies()`
doNotIgnoreValidationError = ignoreRule.applies(); reads almost as a sentence